lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -86,6 +113,14 @@ defmodule TrentoWeb.V1.SapSystemView do
   end
 
   def render("application_instance.json", %{instance: instance}) do
+    instance
+    |> Map.from_struct()
+    |> Map.delete(:__meta__)
+    |> Map.delete(:absent_at)
```

Comment:
We cannot remove the `absent_at`, it is used in the frontend.
And it would look like exactly as the broadcast